system-parachains/asset-hubs/asset-hub-kusama/src/ah_migration/call_filter.rs

```diff
@@ -136,7 +136,7 @@ pub fn call_allowed_status(
 		NftFractionalization(..) => OFF,
 		Recovery(..) => OFF,
 		MultiBlockMigrations(..) => OFF, // has not calls
-		Revive(..) => OFF,               // TODO: OFF or ON?
+		Revive(..) => ON,
```

Comment:
I think contracts should be OFF during the migration. They could interfere with other pallets through XCM Transact.

Reply:
but its still gonna use general executor and filters? but I am gonna disable anyway